### PR TITLE
Fixed login error screens

### PR DESCRIPTION
**Pull request by cecileboucheron**
@keybase/react-hackers

### CODE REVIEW

File: shared/common-adapters/text.meta.desktop.js [hunk 1]
```diff
@@ -225,7 +225,7 @@ const metaData: {[key: TextType]: MetaType} = {
     fontSize: 13,
     styleOverride: {
       ...globalStyles.fontTerminal,
-      lineHeight: '20px',
+      lineHeight: '17px',
```
Comment (chrisnojima) [inline shared/common-adapters/text.meta.desktop.js]: this will change this everywhere, is that what you want?

Comment (cecileboucheron) [inline shared/common-adapters/text.meta.desktop.js]: I just reversed it.

**Comment by keybase-ci-visdiff** [comment]
The commits 77be732043c1...e4807b02f9fe introduce visual changes on linux. <details><summary>:mag_right: 6 new</summary>

 * Added: **Error_View-User_Not_Found**  
   Error_View-User_Not_Found rendered
 * Added: **Error_View-Key_Not_Found**  
   Error_View-Key_Not_Found rendered
 * Added: **Error_View-Key_No_Secret**  
   Error_View-Key_No_Secret rendered
 * Added: **Error_View-Key_No_Matching_GPG**  
   Error_View-Key_No_Matching_GPG rendered
 * Added: **Error_View-Device_No_Provision**  
   Error_View-Device_No_Provision rendered
 * Added: **Error_View-Bad_Login_Password**  
   Error_View-Bad_Login_Password rendered
</summary>

**Comment by keybase-ci-visdiff** [comment]
The commits 77be732043c1...e4807b02f9fe introduce visual changes on linux. <details><summary>:mag_right: 6 new</summary>

 * Added: **Error_View-User_Not_Found**  
   Error_View-User_Not_Found rendered
 * Added: **Error_View-Key_Not_Found**  
   Error_View-Key_Not_Found rendered
 * Added: **Error_View-Key_No_Secret**  
   Error_View-Key_No_Secret rendered
 * Added: **Error_View-Key_No_Matching_GPG**  
   Error_View-Key_No_Matching_GPG rendered
 * Added: **Error_View-Device_No_Provision**  
   Error_View-Device_No_Provision rendered
 * Added: **Error_View-Bad_Login_Password**  
   Error_View-Bad_Login_Password rendered
</summary>

**Pull request merged**